server/schedule/operator/create_operator_test.go

```diff
@@ -233,64 +237,64 @@ func (s *testCreateOperatorSuite) TestCreateMergeRegionOperator(c *C) {
 		},
 	}
 
-	for _, tc := range cases {
-		source := core.NewRegionInfo(&metapb.Region{Id: 68, Peers: tc.sourcePeers}, tc.sourcePeers[0])
-		target := core.NewRegionInfo(&metapb.Region{Id: 86, Peers: tc.targetPeers}, tc.targetPeers[0])
-		ops, err := CreateMergeRegionOperator("test", s.cluster, source, target, 0)
-		if tc.expectedError {
-			c.Assert(err, NotNil)
+	for _, testCase := range testCases {
+		source := core.NewRegionInfo(&metapb.Region{Id: 68, Peers: testCase.sourcePeers}, testCase.sourcePeers[0])
+		target := core.NewRegionInfo(&metapb.Region{Id: 86, Peers: testCase.targetPeers}, testCase.targetPeers[0])
+		ops, err := CreateMergeRegionOperator("test", suite.cluster, source, target, 0)
+		if testCase.expectedError {
+			suite.Error(err)
 			continue
 		}
-		c.Assert(err, IsNil)
-		c.Assert(ops, HasLen, 2)
-		c.Assert(ops[0].kind, Equals, tc.kind)
-		c.Assert(ops[0].Len(), Equals, len(tc.prepareSteps)+1)
-		c.Assert(ops[1].kind, Equals, tc.kind)
-		c.Assert(ops[1].Len(), Equals, 1)
-		c.Assert(ops[1].Step(0).(MergeRegion), DeepEquals, MergeRegion{source.GetMeta(), target.GetMeta(), true})
+		suite.NoError(err)
+		suite.Len(ops, 2)
+		suite.Equal(testCase.kind, ops[0].kind)
+		suite.Equal(len(testCase.prepareSteps)+1, ops[0].Len())
```

Comment:
use `suite.Len`?

Reply:
What about:

```suggestion
		suite.Len(testCase.prepareSteps, ops[0].Len()-1)
```

Reply:
Is it reasonable to call `Len` for expected value?

Reply:
OK. You're right.